src/clip.rs

```diff
@@ -29,10 +29,9 @@ pub fn copy_to_clipboard(s: &SafeString) -> Result<(), ()> {
 
 // On UNIX, the most stable way to copy to the clipboard is using one of the existing
 // and battle tested tools: xsel and xclip.
-#[cfg(all(unix, not(macos)))]
+#[cfg(all(unix, not(macos), not(windows)))]
```

Comment:
What does this do ? Given that I have selected `unix` and Windows is not a UNIX, I feel like this change is useless. What do you think ?

Reply:
For whatever reason cargo tries to compile that function unless I specifically tell it not too.

Reply:
Hmm, alright, odd, well then feel free to let this here, I don't mind, just seems odd